app/src/main/java/com/pvp/app/model/Tasks.kt

```diff
@@ -173,13 +180,37 @@ open class Task(
         return "Task(date=$date, description=$description, duration=$duration, id=$id, isCompleted=$isCompleted, points=$points, time=$time, title='$title', userEmail='$userEmail')"
     }
 
+    fun getNotification(): Notification? {
```

Comment:
This is a model / entity, it should not contain such logic here. It _can_ stay here, but it _should not_. This is more of a service functionality, which defines how Notification is resolved from a task and how Notification is defined (text, title).